flink-filesystems/flink-s3-fs-base/src/test/java/org/apache/flink/fs/s3/common/MinioTestContainer.java

```diff
@@ -113,16 +114,16 @@ private String getHttpEndpoint() {
      * relevant parameter to access the {@code Minio} instance.
      */
     public void setS3ConfigOptions(Configuration config) {
-        config.setString(FLINK_CONFIG_S3_ENDPOINT, getHttpEndpoint());
+        config.set(AbstractS3FileSystemFactory.ENDPOINT, getHttpEndpoint());
```

Comment:
nit: re-order commits so that the use of options goes after their introduction?

Currently, I see it as
```
[hotfix] Use newly defined ConfigOptions in MinioTestContainer 
[hotfix] Move CompressionUtils to flink-core
[hotfix] Create ConfigOptions for s3 access/secret keys and endpoint 
```